index.rst

```diff
@@ -62,6 +62,49 @@ These tools are intended for use by the full range of geosciencies community:
 beginners and experts; students and teachers; national centers and university
 research labs.
 
+Nonlinear and Non-Gaussian Data Assimilation Capabilities in DART
+-----------------------------------------------------------------
+The default DART algorithms assume a normal distribution to compute ensemble increments
+for the observed quantity (this is the ensemble adjustment Kalman filter, or EAKF) and 
+then linearly regresses the observation increments onto each state variable.
+ 
+DART’s newest and innovative capability, the Quantile Conserving Filters (QCF), also known
```

Comment:
I think we should leave this as well for another pull request, since we'll need to change a bunch of file names, code, documentation, and update all the model input.nml files